incentivizing_rewarding_participants.adoc

```diff
@@ -1,87 +1,129 @@
+More important is understanding how participants' contributions align with or reflect their motivations for _making_ those contributions—and responding accordingly.
+As https://opensource.com/article/17/10/managing-casual-contributors[researcher Ann Barcomb writes], "People who contribute to boost their CV or to fix an irritating bug are less likely to continue once their need is satisfied."
+Alternatively, however, these periodic contributors (or what Barcomb calls "episodic contributors") often seek low-complexity, easily explainable, and quickly addressable issues—the very kinds of issues long-time contributors would love to see others solve so they can focus on other matters.
+Community managers therefore shouldn't strive to create experiences _they believe_ will motivate everyone to join and contribute to the project as much as they should work to understand participants' _own_ motivations for contributing to projects, then architecting project experiences that activate and satisfy those motivations, leading (presumably) to greater engagement.
+This is critical not only because it puts community members and their needs first, but also because it allows contributors with varying degrees of time, resources, knowledge, and skill to engage a project in ways that feel comfortable for them—and feel rewarded in just the same ways others might.
+
+== Understanding a project's reward models
+
+Strategies for acknowledging and honoring various contributions to an open source project are called that project's "reward models."
+The more reward models your project constructs, the better it can engage participants with differing motivations for contributing.
+As we've already noted, everyone contributes to projects for different reasons, and the most popular projects are those that make contributors feel rewarded, valued, and respected for their contributions—whatever form they happen to take.
+Generating those feelings in contributors is a great way to build your project's reputation as a rewarding, welcoming place.
+But always remember that your community's reward models will _incentivize_ the very practices they _reward_; when people understand how a community recognizes and honors its participants, they'll engage in the behaviors they observe a community celebrate in this way.
+Community managers and architects should therefore always think carefully and act deliberately when establishing and modifying a project's particular reward model.
+What they reward—either intentionally or unintentionally—is what they'll be encouraging people to do more of.
+
+In what remains of this chapter, we'll outline some of the more popular reward models open source projects use today.
+As each of these models aims to connect with a certain set of motivations participants may have for contributing to a project, we'll elaborate situations in which a particular model might be most useful.
+
+=== Point, badge, and leaderboard systems
```

Comment:
I'd like to argue that point systems and leaderboards should be a separate section from badging altogether. It is just so different from the former two.

Badging may be applied to projects, events, individuals, organizations, etc. and it has completely different roots.

Reply:
This is a very good point and it dovetails nicely with the resource you shared in the previous comment. So I will do my best to extricate.